WooCommerce/Classes/ViewRelated/Products/Edit Product/ProductFormViewController.swift

```diff
@@ -735,6 +731,9 @@ private extension ProductFormViewController {
     func observeUpdateBlazeEligibility() {
         blazeEligibilitySubscription = viewModel.blazeEligibilityUpdate.sink { [weak self] in
             self?.updateFormTableContent()
+            if self?.viewModel.canPromoteWithBlaze() == true {
+                ServiceLocator.analytics.track(event: .Blaze.blazeEntryPointDisplayed(source: .productDetailPromoteButton))
+            }
```

Comment:
Similar as above, we should track the tap event with this source only if the intro view will not be displayed:

```suggestion
            guard let self else { return }
            self.updateFormTableContent()
            if self.viewModel.canPromoteWithBlaze() && !self.viewModel.shouldShowBlazeIntroView {
                ServiceLocator.analytics.track(event: .Blaze.blazeEntryPointDisplayed(source: .productDetailPromoteButton))
            }
```